ui/src/main/webapp/lib/admin-wizard/inputs.js

```diff
@@ -71,14 +70,19 @@ const Hostname = ({ label = 'Hostname', ...rest }) => (
   <Input label={label} {...rest} />
 )
 
-const InputAutoView = ({ value = '', options = [], type = 'text', message = {}, onEdit, label, tooltip, ...rest }) => {
+const InputAutoView = ({ value = '', options = [], type = 'text', message = {}, onEdit, label, tooltip, muiTheme, ...rest }) => {
   const { type: messageType, message: text } = message
   for (let key in rest) {
     if (rest[key] === undefined) {
       delete rest[key]
     }
   }
 
+  const messageStyles = {
```

Comment:
They both need `muiTheme` which is only available within the context of these individual components. I could extract them out to a common method that they both can use but that method would need to be passed both the SUCCESS/WARNING key as well as the muiTheme object itself since the theme object is dynamically changed during runtime.